ddtrace/monkey.py

```diff
@@ -200,7 +201,15 @@ def patch(raise_errors=True, **patch_modules):
     )
 
 
+@deprecated(
+    message="This function will be removed.",
+    version="1.0.0",
+)
 def patch_module(module, raise_errors=True):
```

Comment:
we should keep the `# type: (str, bool) -> bool` comment here just like it was before (like how it is for `_patch_module` right now)